processing/src/main/java/io/druid/query/QueryContexts.java

```diff
@@ -37,6 +37,8 @@
   public static final boolean DEFAULT_BY_SEGMENT = false;
   public static final boolean DEFAULT_POPULATE_CACHE = true;
   public static final boolean DEFAULT_USE_CACHE = true;
+  public static final boolean DEFAULT_POPULATE_RESULTLEVEL_CACHE = true;
```

Comment:
New feature should start off disabled

Reply:
Would it be sufficient to keep it disabled via the populateResultCache parameter?
https://github.com/druid-io/druid/pull/5028/commits/ba78816e73e0645ddd537b52a1a77e25d3f5855a#diff-661a9dbd25633c651845a9ecab0ddbaaR44

Reply:
It is confusing to have default here be `true` but the default in `CacheConfig` be false. But this just follows the convention of the prior caching settings.

Can you add a comment in `CacheConfig` that the defaults as stated in `QueryContexts` are different due to legacy reasons, and should probably be made the same at some point in the future?

Reply:
Added comment as suggested.